style.css

```diff
@@ -0,0 +1,784 @@
+  color: #fff;
+  font-size: 2rem;
+  margin-bottom: 0;
+  text-align: center;
+  padding-bottom: 0.5rem;
+}
+
+.program-separator {
+  margin: 0 auto;
+  height: 0.1rem;
+  width: 5%;
+  border-bottom: 2px solid var(--orange-color);
+  margin-bottom: 2rem;
+}
+
+.course .image-container {
+  width: 30%;
+}
+
+.logo-image img {
+  width: 100%;
+  max-width: 300px;
+}
+
+.course .image-container img {
+  width: 50%;
+  padding-left: 1rem;
+  object-fit: fill;
+}
+
+.course {
+  padding: 0.5rem;
+  display: flex;
+  align-items: center;
+  gap: 1rem;
+  margin: 0.5rem;
+  border-radius: 5px;
+  background: rgba(255, 255, 255, 0.1);
+}
+
+.course:hover {
+  border: 1px solid #fff;
+}
+
+.course-title {
+  font-family: var(--lato-medium);
+  color: var(--orange-color);
+  font-size: 1.5rem;
+  width: 30%;
+}
+
+.course-description {
+  font-family: var(--lato);
+  color: #fff;
+  width: 60%;
+}
+
+#join-program {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  margin: 2rem 0;
+}
+
+#join-button {
+  font-family: var(--lato-medium);
+  padding: 1.5rem;
+  font-size: 1.5rem;
+  color: #fff;
+  background-color: var(--orange-color);
+  border: none;
+}
+
+#join-button:hover {
+  background-color: #fff;
+  color: var(--orange-color);
+  transform: scale(1.1);
+}
+
+#join-link {
+  display: none;
+}
+
+#featured-mentors {
+  margin-bottom: 2rem;
+}
+
+.featured-mentors-title {
+  margin-top: 4rem;
+  font-family: var(--lato-medium);
+  font-size: 2rem;
+  margin-bottom: 0;
+  text-align: center;
+  padding-bottom: 0.5rem;
+}
+
+.featured-mentors-separator {
+  margin: 0 auto;
+  height: 0.1rem;
+  width: 10%;
+  border-bottom: 2px solid var(--orange-color);
+  margin-bottom: 2rem;
+}
+
+.mentor-card {
+  padding: 0 1rem;
+}
+
+.mentor-card .image-container {
+  float: left;
+  width: 150px;
+  height: 180px;
+  margin: 0.2rem;
+  border-image: url('./assets/images/pngfind.com-border-png-607777.png') 5%;
+  border-style: solid;
+  border-width: 15px;
+}
+
+.mentor-card .image-container img {
+  width: 100%;
+  height: 100%;
+}
+
+.mentor-name {
+  font-family: var(--lato-medium);
+  font-weight: 600;
+  font-size: 1.3rem;
+  padding-top: 0.5rem;
+  margin: 0;
+}
+
+.mentor-title {
+  color: var(--orange-color);
+  font-family: var(--lato-medium);
+  font-weight: 600;
+  font-size: 1.1rem;
+  margin: 0;
+}
+
+.mentor-separator {
+  float: left;
+  height: 0.1rem;
+  padding-left: 0.5rem;
+  border-radius: 5px;
+  width: 5%;
+  border-bottom: 5px solid var(--light-background);
+}
+
+.mentor-description {
+  font-family: var(--lato);
+  font-size: 1rem;
+}
+
+.mentors-container {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-row-gap: 3rem;
+}
+
+#sponsor {
+  background-color: var(--dark-background);
+  padding-bottom: 2rem;
+  padding-top: 0.5rem;
+}
+
+.sponsor-row {
+  padding: 1rem 0 1rem 0;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+}
+
+footer {
+  width: 100%;
```

Comment:
- [ ] You have some gap to the left of your page and is caused by this width, if you delete it everything will be fine ;😁 😁 

<img width="781" alt="Screen Shot 2023-02-07 at 23 37 48" src="https://user-images.githubusercontent.com/74501523/217360327-32fdb158-f64d-44c0-b5e9-700420cd3fba.png">